donut-app/source/StoreFinder.mc

```diff
@@ -0,0 +1,10 @@
+
+class StoreFinder {
+	function registerSetters(textSetter, metersSetter, isOpenSetter) {
+	
+	}
+
+	function requestNearestStore() {
```

Comment:
Add a parameter: `posDegrees` which is the position in degrees of the location we are at.